src/views/GraphDetail.vue

```diff
@@ -382,6 +383,9 @@ export default Vue.extend({
     drawerIcon(): string {
       return this.panelOpen ? 'expand_more' : 'expand_less';
     },
+    userToken() {
```

Comment:
Rename this to `loginToken()` to reflect @AlmightyYakob's comments above.

Reply:
See 89f51e5